mediator/med.F90

```diff
@@ -835,14 +851,46 @@ subroutine InitializeIPDv03p1(gcomp, importState, exportState, clock, rc)
 
     if (mastertask) then
        write(logunit,*)
```

Comment:
I really think that you missed my point here.  If the component is present print the name of the component, if no component is present don't print anything at all.  What you have is worse than what you are replacing.

Reply:
No, I understand. Thanks.